README.md

```diff
@@ -0,0 +1,21 @@
+# foo-config-loader
+
+A config loader for the Foo project.
+
+
+# Project tasks
+
+## In order to setup a local env run...:
+
+    virtualenv create -p python3.6 venv/
```

Comment:
I think that `virtualenv` is deprecated since `python3.3` (see: https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#installing-virtualenv ) how about using `python3.6 -m venv` instead?